### PR TITLE
💄 add a ramps look & feel for custom add extract commons methods

### PR DESCRIPTION
**Pull request by coliss86**
New look of custom robot:
image

Based on the work in #557

### CODE REVIEW

File: src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/Makelangelo5.java [hunk 1]
```diff
@@ -2,18 +2,13 @@
...
-import com.jogamp.opengl.util.texture.TextureIO;
-import com.marginallyclever.convenience.FileAccess;
 import com.marginallyclever.convenience.Point2D;
 import com.marginallyclever.makelangelo.plotter.Plotter;
 import com.marginallyclever.makelangelo.plotter.settings.PlotterSettings;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
+import static com.marginallyclever.makelangelo.plotter.plotterRenderer.DrawingAssistant.*;
```
Comment (i-make-robots) [inline src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/Makelangelo5.java:9]: I'm dumb.  why 'static'?

File: src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/MakelangeloCustom.java [hunk 2]
```diff
@@ -253,141 +213,9 @@ private void paintPenHolderToCounterweights( GL2 gl2, Plotter robot ) {
...
-		gl2.glVertex2d(right, top);
-
-		gl2.glVertex2d(right, top);
-		gl2.glVertex2d(right-side_crit, bot_crit);
-
-		gl2.glEnd();
-		drawArc(gl2, (float)right, (float)top, (float)belt, (float)(Math.PI*1.5-sideang), (float)(Math.PI*1.5-botang));
-		drawArc(gl2, (float)left, (float)top, (float)belt, (float)(Math.PI*1.5+botang), (float)(Math.PI*1.5+sideang));
-	}
-/*
```
Comment (i-make-robots) [inline src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/MakelangeloCustom.java:328]: i am keeping these until we have templates setup for each machine style.  
Then these number can be copied into the template.  Same is true for other machine types.

File: src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/DrawingAssistant.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+import com.jogamp.opengl.util.texture.Texture;
+import com.jogamp.opengl.util.texture.TextureIO;
+import com.marginallyclever.convenience.FileAccess;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+
+public class DrawingAssistant {
```
Comment (i-make-robots) [inline src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/DrawingAssistant.java:13]: this should probably be part of com.marginallyclevver.convenience.OpenGLHelper or maybe com.marginallyclevver.convenience.DrawingHelper

File: src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/Makelangelo5.java (hunk 1 above)
Comment (coliss86) [reply src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/Makelangelo5.java:9]: `import static` enables you to call method of `DrawingAssistant` like they were defined in your class or a parent.
`Makelangelo5.texture1 = loadTexture("/textures/makelangelo5.png");` <= note here that the call to`loadTexture()` is not `DrawingAssistant.loadTexture()` but just `loadTexture()`
Another example is in unit test where methods like `assertEquals() came from an `import static org.junit.jupiter.api.Assertions.assertEquals;`

File: src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/DrawingAssistant.java (hunk 3 above)
Comment (coliss86) [reply src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/DrawingAssistant.java:13]: ok moved to `com.marginallyclevver.convenience.DrawingHelper`

File: src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/MakelangeloCustom.java (hunk 2 above)
Comment (coliss86) [reply src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/MakelangeloCustom.java:328]: ok I put this back

**Comment by i-make-robots** [comment]
Looking good, good lookin'!

**Pull request merged**